index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/a12691952c.js" crossorigin="anonymous"></script>
+    <title>Portfolio David Tamayo</title>
+</head>
+<body>
+    <div class="mainpage">
+        <header class="toolbar">
+            <h2 class="welcome">Welcome</h2>
+            <button class="vector">☰</button>
+        </header>
+        <section class="headline">
```

Comment:
- [ ] Please use for this section the proper background image used for this template. If you notice that an image is not part of the content (aesthetic reasons) use a CSS background image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info. :+1: 
Please check the screenshots below to check the difference :arrow_down: 

#### You design
![Screenshot from 2022-08-18 01-14-07](https://user-images.githubusercontent.com/6615394/185272966-e071c042-9166-4ae4-8853-7b8c1fc071df.png)

#### Figma Design
![Screenshot from 2022-08-18 01-13-30](https://user-images.githubusercontent.com/6615394/185272970-89de0580-2b7c-42d0-9ce4-f3a3565790ae.png)

- [ ] Please make sure that you use the font and colors (background and font) defined for the template. :+1: 

* If it's pretty and works it will be a wonderful website.

Reply:
Hi @codecaiine I have a problem with this because in my computer there is no diference. I used it as a CSS background image.
![Captura de pantalla de 2022-08-18 09-15-43](https://user-images.githubusercontent.com/105117832/185418455-b8b874d2-719f-46d8-8e35-369e6177584a.png)

Reply:
Hi, @IngDavidTM I understand! I noticed that on the screenshot you shared with me you are using the desktop version. I would like to recommend you adjust your design for a Minimum mobile screen size that should be 375px with media query It will work :+1: